code/spring-boot-telemetry/src/main/resources/application.properties

```
@@ -1,14 +1,16 @@
-
 spring.application.name=SpringVeggieService
 spring.datasource.name=veggies
+spring.datasource.url=jdbc:otel:postgresql://db:5432/demo
+spring.datasource.driver-class-name=io.opentelemetry.instrumentation.jdbc.OpenTelemetryDriver
 
-spring.datasource.url=jdbc:postgresql://db:5432/demo
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=create
 
 spring.datasource.username=demo
 spring.datasource.password=demo
 
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.open-in-view=false
 
+applicationinsights.connection.string=InstrumentationKey=yourInsightsKey
```

To have the same thing for Spring and Quarkus:

```suggestion
applicationinsights.connection.string=InstrumentationKey=00000000-0000-0000-0000-000000000000
```